app/forms/hyrax/forms/resource_batch_edit_form.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+module Hyrax
+  module Forms
+    class ResourceBatchEditForm < Hyrax::Forms::ResourceForm
+      include Hyrax::FormFields(:basic_metadata)
+
+      self.required_fields = []
+      self.model_class = Hyrax.primary_work_type
+
+      # Contains a list of titles of all the works in the batch
+      attr_accessor :names
+
+      # @param [Hyrax::Work] model the model backing the form
+      # @param [Ability] current_ability the user authorization model
+      # @param [Array<String>] batch_document_ids a list of document ids in the batch
+      def initialize(model, _current_ability, batch_document_ids)
+        @names = []
+        @batch_document_ids = batch_document_ids
+        if @batch_document_ids.present?
+          super(model.class.new(initialize_combined_fields))
+        else
+          super(model)
+        end
+      end
+
+      def terms
```

Comment:
why is it okay to hard code these values? shouldn't they be derived from configuration?

Reply:
Action item: See if we can use the model configurable fields. Pull @no-reply in if we get stuck with this.